tests/library/browsercontext-reuse.spec.ts

```diff
@@ -17,17 +17,19 @@
 import { browserTest, expect } from '../config/browserTest';
 import type { BrowserContext, BrowserContextOptions } from '@playwright/test';
 
-const test = browserTest.extend<{ reusedContext: (options?: BrowserContextOptions) => Promise<BrowserContext> }>({
-  reusedContext: async ({ browserType, browser }, use) => {
+const test = browserTest.extend<{}, { reusedContext: (options?: BrowserContextOptions) => Promise<BrowserContext> }>({
+  reusedContext: [async ({ browserType, browser }, use) => {
+    let context;
     await use(async (options: BrowserContextOptions = {}) => {
       const defaultContextOptions = (browserType as any)._defaultContextOptions;
-      const context = await (browser as any)._newContextForReuse({
+      context = await (browser as any)._newContextForReuse({
         ...defaultContextOptions,
         ...options,
       });
       return context;
     });
-  },
+    await context.close();
```

Comment:
We don't need this one anymore, do we?

Reply:
This would otherwise keep a context lying around after the last test of this file has been completed.

Reply:
Since this file is testing a reused browser context, perhaps we should keep it open to be reused? Anyway, this seems like an unrelated fix to me.